src/Notepads/Controls/FindAndReplace/FindAndReplaceEventArgs.cs

```diff
@@ -9,16 +9,23 @@ public enum FindAndReplaceMode
         ReplaceAll
     }
 
+    public enum Direction
```

Comment:
Rename it to SearchDirection and use "Previous" or "Next" here. The reason we should not use down or up is because we are assuming text flow is always top to down which might not be true for some languages. I could not think of any but there is a potential, similar like Right to Left languages. Previous or Next seems more accurate here.